security/tinc/src/opnsense/scripts/OPNsense/Tinc/lib/objects.py

```diff
@@ -97,6 +95,10 @@ def set_PMTUDiscovery(self, value):
     def config_text(self):
         result = list()
         result.append('AddressFamily=any')
+        if 'address' in self._payload:
+            addresses = self._payload['address'].split(',')
+            for address in addresses:
```

Comment:
can be simplified:

```
for address in self._payload['address'].split(','):
```

`addresses`  isn't used anywhere else.

It's probably better to move the functionality to `NetwConfObject. get_addresses()` in which case you could iterate always over the get_addresses() and ditch the `if` as well

Reply:
Was micmic of existing code for subnets...
Reworked both this way.

Reply:
Did so, but this part shouldn't have been, anyway...
Undone. Took care of your comment in next code block.